test/functional/inspec_exec_test.rb

```diff
@@ -301,7 +301,8 @@
   describe 'when a dependency does not support our backend platform' do
     it 'skips the controls from that profile' do
       out = inspec("exec #{File.join(profile_path, 'profile-support-skip')} --no-create-lockfile")
-      out.stdout.force_encoding(Encoding::UTF_8).must_include "Profile Summary: 0 successful controls, 0 control failures, \e[38;5;247m2 controls skipped\e[0m\n"
+      out.stdout.force_encoding(Encoding::UTF_8).must_include "WARN: Skipping profile: 'windows-only' on unsupported platform"
```

Comment:
Personally, I prefer error messages that are telling me I did something wrong, to give hints as to how to do it right; here, you could add info as to the current platform (platform lineage?), and a list of the supported platforms that the profile supports.

EDIT: ahh, I see you do include the backend platform.  Mentioning the profile platform might be nice, though.  Not a blocker.